lib/convection/control/cloud.rb

```diff
@@ -114,6 +114,18 @@ def diff(to_stack, options = {}, &block)
           return
         end
 
+        errors = []
+        stacks.each_value do |stack|
+          if stack.error?
+            errors << stack.errors.collect { |x| x.exception.message }
+          end
+        end
+        errors = errors.uniq.flatten!
+        unless errors.empty?
+          block.call(Model::Event.new(:error, "Error(s) during stack diff #{errors.join(', ')}", :error), errors) if block
+          return
+        end
```

Comment:
1. Should this be within `filter_deck` below so that we only throw errors on the current stack-group or filtered stacks?
2. Should we do one `block.call` per stack to more closely follow how we handle errors in the `converge` method?
3. Should we proceed with the diff if an error occurs?  Do we exit with a non-zero exit status when errors occur?